src/BenchmarksApps/HttpClientBenchmarks/Clients/HttpClient/ClientOptions.cs

```diff
@@ -18,6 +18,7 @@ public class ClientOptions
     public int ContentWriteSize { get; set; }
     public bool ContentFlushAfterWrite { get; set; }
     public bool ContentUnknownLength { get; set; }
+    public string[]? Header { get; set; }
```

Comment:
RootCommand can automatically assign only the arguments of the same name, e.g. if I rename it to "Headers" here, I would need to pass them from CLI as `--headers "bla: bla"`. I was lazy to write a custom mapper to ClientOptions 😄 but it might make sense now. I guess parsing headers into key-value pairs can also become part of that.

Reply:
But it would look so ugly 😢

Reply:
How does the command like for multiple values look like exactly? Is it `--header a --header b` or `--header a b`?
I'm looking at https://github.com/dotnet/command-line-api/blob/main/docs/model-binding.md#arrays-lists-and-other-enumerable-types and trying to understand how all this work.

Reply:
`--header a --header b` . It might also work with  `--header a b` but only if  `Option` is specifically configured. But first way solved our problem nicely as is so I didn't explore the second one

Reply:
I'm prototyping the binder btw.

Reply:
Binder is in. Also updated System.CommandLine to beta.2 (they had breaking changes :D)